site/server/test/UserStoreSpec.scala

```diff
@@ -0,0 +1,103 @@
+import org.specs2.mutable.Specification
+import org.specs2.specification.BeforeEach
+import test.database.TestDatabase
+import doobie.imports._
+import scalaz.concurrent.Task
+import shared.User
+import com.fortysevendeg.exercises.models.UserDoobieStore
+
+class UserStoreSpec extends Specification with BeforeEach {
+  val transactor: Transactor[Task] = TestDatabase.transactor
+  import transactor.yolo._
+
+  def before = UserDoobieStore.deleteAll.quick.run
+
+  "UserDoobieStore" should {
+    "create a new User and retrieve it afterwards" in {
+      val aUser = User(None, "47deg", "47 degrees", "47deg", "http://placekitten.com/50/50", "http://github.com/47deg", "hi@47deg.com")
```

Comment:
Same line in several places in this file. Could we extract this kind of code in factory method? We could receive all the parameters but with the less important ones with default values?